sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/ContinuousExecution.scala

```diff
@@ -251,12 +251,8 @@ class ContinuousExecution(
 
       updateStatusMessage("Running")
       reportTimeTaken("runContinuous") {
-        SQLExecution.withNewExecutionId(
-          sparkSessionForQuery, lastExecution) {
-          // Materialize `executedPlan` so that accessing it when `toRdd` is running doesn't need to
-          // wait for a lock
-          lastExecution.executedPlan
-          lastExecution.toRdd
+        SQLExecution.withNewExecutionId(sparkSessionForQuery, lastExecution) {
+          lastExecution.executedPlan.execute()
```

Comment:
Looks good as it really just skip the assignment of the lazy val and uses the value of the right side of the assignment:

https://github.com/apache/spark/blob/07b79c58618488ced10e7c29b78eaca1a64328e2/sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala#L98